openstack/blockstorage/storagepools/results.go

```diff
@@ -0,0 +1,122 @@
+package storagepools
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+
+	"github.com/gophercloud/gophercloud"
+)
+
+type ListResult struct {
+	gophercloud.Result
+}
+
+// Minimum set of driver capabilities only
+// https://github.com/openstack/cinder/blob/master/cinder/interface/volume_driver.py#L56
+type Capabilities struct {
+	// Required Fields
+	DriverVersion     string  `json:"driver_version"`
+	FreeCapacityGB    float64 `json:"-"`
+	StorageProtocol   string  `json:"storage_protocol"`
+	TotalCapacityGB   float64 `json:"-"`
+	VendorName        string  `json:"vendor_name"`
+	VolumeBackendName string  `json:"volume_backend_name"`
+	// Optional Fields
+	ReservedPercentage       int64   `json:"reserved_percentage"`
+	LocationInfo             string  `json:"location_info"`
+	QoSSupprt                bool    `json:"QoS_support"`
+	ProvisonedCapacityGB     float64 `json:"provisoned_capacity_gb"`
```

Comment:
Haven't looked at the schema, but I suspect these should be "Provisioned" (and "provisioned")